alternatives.c

```diff
@@ -255,6 +255,27 @@ char *parseLine(char **buf) {
     return strdup(start);
 }
 
+void clean_path(char *path, char c) {
```

Comment:
I can't imagine that we would like to remove anything else than /. I would drop the char and hardcode '/'

Reply:
True, thank you.